easybuild/easyconfigs/p/PyTorch/PyTorch-2.1.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -171,7 +171,7 @@ dependencies = [
     ('expecttest', '0.1.5'),
     ('networkx', '3.1'),
     ('sympy', '1.12'),
-    ('Z3', '4.12.2'),
+    ('Z3', '4.12.2', '-Python-%(pyver)s', ('GCCcore', '12.3.0')),
```

Comment:
This was taken out on purpose, see https://github.com/easybuilders/easybuild-easyconfigs/pull/20050
We now have _one_ `Z3` which has Python support. Ie. no more build with/without suffix. Please undo this change. You can then build your `torchvision` against PR 20050.